### PR TITLE
Makes a Couple Minor PR Template Improvements

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Formatting was whack

### How do I test this PR?

No testing necessary. Do you like how this looks???

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

---

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](url)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,17 +4,19 @@
...
+---
+
 > I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
 
-## TODO:
+## TODO
 
 - [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
-- [ ] Closes [#ISSUE]
+- [ ] Closes [DEV-XXX](url)
```
Comment (richarddubay) [inline .github/PULL_REQUEST_TEMPLATE.md]: This creates the `DEV-XXX` as a link that doesn't go anywhere. Maybe change the formatting so that it doesn't create the URL but it does suggest that what you're looking for here is a URL to the Jira issue. 

Also, I just wanted to note that when you put in a Github issue number, this actually closes the Github issue. This probably doesn't _actually_ move the Jira issue to `Done` or anything. Man ... if only it could. I get the idea though ... just putting in a reference to the task this PR was for is smart. 👍

**Review by richarddubay — CHANGES_REQUESTED** [review]

**Comment by redreceipt** [comment]
@richarddubay I put `#url` in the template so it doesn't actually create a link but still shows you to put the url in there.

**Review by richarddubay — APPROVED** [review]
❤️

**Comment by redreceipt** [comment]
[asdf](https://trello.com/c/TH7FS3MC/2-asdf)

**Review by cdanwards — APPROVED** [review]
